__init__.py

```diff
@@ -57,14 +57,15 @@ def get_proj(path):
             if o.attrib["name"] == 'additionalInfo':
                 add_info = o[0]
 
-    if len(items) == 0:
-        return []
-
     if add_info is not None:
         for i in add_info:
             for o in i[0][0]:
                 if o.tag == 'option' and 'name' in o.attrib and o.attrib["name"] == 'projectOpenTimestamp':
                     items[i.attrib["key"]] = int(o.attrib["value"])
+
+    if len(items) == 0:
+        return []
+
```

Comment:
Can't we just leave it out entirely? the following line will return an empty array regardless, doesn't it?

Reply:
Honestly, I don't know python at all, so i'm just pretty much shuffling stuff around and guessing what it does until it starts  working :)
I wasn't sure what the last return statement would do if items were empty since I'm not familiar with the syntax

Reply:
I tried it out and it works just fine without the
```python
    if len(items) == 0:
        return []
```
lines. I'll merge this and clean up afterwards :)